vector/src/main/java/im/vector/app/features/home/room/detail/timeline/factory/MessageItemFactory.kt

```diff
@@ -168,6 +171,11 @@ class MessageItemFactory @Inject constructor(
             return noticeItemFactory.create(params)
         }
 
+        // This is a thread event and we will [debug] display it when we are in the main timeline
```

Comment:
I would have move the comment inside the `if` block.

Reply:
True, updated